administrator/components/com_cookiemanager/src/Table/GroupTable.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_cookiemanager
+ *
+ * @copyright   (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Contact\Administrator\Table;
+
```

Comment:
namespace Contact?

Reply:
corrected  thank you
Joomla does a lot of magic behind the scenes using namespace